src/main/resources/assets/extraalchemy/lang/pt_br.json

```diff
@@ -154,6 +154,14 @@
 	"extraalchemy.config.general.enable_brewing_stand_fire.tooltip1": "Habilitar isto faz com que o alambique seja reabastecido a cada", 
 	"extraalchemy.config.general.enable_brewing_stand_fire.tooltip2": "2 segundos quando uma fonte de calor for colocada abaixo dele.",
 	
+	"extraalchemy.config.general.brewing_stand_heat_increment_delay": "Brewing stand heat accumulation speed",
```

Comment:
Just remove these lines completely, they will automatically fall back to English if missing